Source/ExcelDna.sln

```diff
@@ -1,6 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
+# Visual Studio 14
```

Comment:
Nothing really "depends" on this change for it all to work, but this makes the solution compatible with Visual Studio 2015, whilst keeping compatibility to previous version - i.e. `MinimumVisualStudioVersion = 10.0.40219.1` so it's harmless.